webdriver/tests/bidi/script/add_preload_script/contexts.py

```diff
@@ -68,18 +68,46 @@ async def test_page_script_context_isolation(bidi_session, add_preload_script,
         wait="complete",
     )
 
-    # Check that preload script applied the changes to the window
+    # Check that preload script applied the changes to the context
     result = await bidi_session.script.evaluate(
         expression="window.baz",
         target=ContextTarget(top_context["context"]),
         await_promise=True,
     )
     assert result == {"type": "number", "value": 42}
 
-    # Check that preload script did *not* apply the changes to the other window
+    # Check that preload script did *not* apply the changes to the other context
     result = await bidi_session.script.evaluate(
         expression="window.baz",
         target=ContextTarget(new_context["context"]),
         await_promise=True,
     )
     assert result == {type: "undefined"}
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("domain", ["", "alt"], ids=["same_origin", "cross_origin"])
+async def test_identical_contexts(
```

Comment:
Is there a reason why we need to test iframes here? We can only pass contexts for top-level browsing contexts anyway.

Reply:
No, changed.